itis.playground/Pages/homework.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,677 @@
+//: [Previous](@previous)
+
+import Foundation
+
+var greeting = "Hello, playground"
+
+//task 2
+
+//Absrtaction
+
+protocol Human{
+    var name: String? {get set}
```

Comment:
```suggestion
    var name: String { get set }
```
1. стиль
2. почему имя опциональное? У человека не может быть имени?